doc/self-hosting.md

```diff
@@ -184,9 +184,14 @@ Start the server using the Sentry DSN. You can set it:
 sudo SENTRY_DSN=https://xxx:yyy@sentry.io/zzz node server
 ```
 
-- or by `sentry_dsn` secret property defined in `private/secret.json`
+Or via [checked in config](server-secrets.md):
```

Comment:
Coming at this cold but I was thrown off by the `checked in..` wording. Is my understanding correct that the intent is to point folks to the server-secrets file for more info about how configs can be set, or encouraging them to check something in, in the VCS sense?

Reply:
Yes, exactly. Maybe "checked-in config" isn't the best way to phrase it. Open to better wording…

Reply:
Maybe `Or via config, as you would do with [server secrets](server-secrets.md):`. If they don't know how to set secrets via config, they can go to that documentation where there's a little bit more detail.